src/cmd/tools.go

```diff
@@ -121,13 +121,13 @@ var generatePKICmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		pki := pki.GeneratePKI(args[0], subAltNames...)
 		if err := os.WriteFile("tls.ca", pki.CA, 0644); err != nil {
-			message.Fatalf(err, lang.ErrWritingFile, "tls.ca", err)
```

Comment:
Was there a particular reason to change these from error unwrapping (as we do most places now) to simple string?

Reply:
Unit tests will fail since that method doesn't support error unwrapping...

![image](https://user-images.githubusercontent.com/3977569/207169897-723d7565-6af1-4c6e-ad23-b537af73cf12.png)

(Pretty much) everywhere else is a fmt.Errorf function whereas this is the final message.Fatalf)

Reply:
Technically since they are in a different folder it would work now, but I felt it was better to correct those.